### DISCUSSION TITLE
Error adminjs with typescript

### DISCUSSION THREAD

**Discussion by AndreFelipe97** [SoftwareBrothers/admin-bro#1548] [General]
I have an error when starting the project

TS config

![image](https://github.com/SoftwareBrothers/adminjs/assets/38105957/91dc741f-1e41-44c7-b232-c27513783276)

package json

![image](https://github.com/SoftwareBrothers/adminjs/assets/38105957/8985ee3e-3540-4c31-8ab4-8563e9b6a47d)

Console Error

![image](https://github.com/SoftwareBrothers/adminjs/assets/38105957/748a7522-1920-4563-8539-daaa12187dea)

**Comment by luiskurihara** [comment]
Fala felipe estou com o mesmo erro!!! ja conseguiu resolver??

**Comment by AshotN** [comment]
AdminJS doesn't support CJS anymore, you need to switch over to ESM. Or downgrade to adminjs v6
